regression/cbmc/Pointer_byte_extract5/no-simplify.desc

```diff
@@ -1,4 +1,4 @@
-CORE
+CORE broken-smt-backend
```

Comment:
:no_entry_sign: Please add an explanation / link to relevant issue as to why this is broken with smt-backend at the bottom of the desc file

Reply:
Added.